plexapi/audio.py

```diff
@@ -115,7 +115,7 @@ def sync(self, bitrate, client=None, clientId=None, limit=None, title=None):
 
 @utils.registerPlexObject
 class Artist(Audio, ArtMixin, PosterMixin, SplitMergeMixin, UnmatchMatchMixin,
-        CollectionMixin, CountryMixin, GenreMixin, MoodMixin, SimilarArtistMixin, StyleMixin):
+        CollectionMixin, CountryMixin, GenreMixin, MoodMixin, SimilarArtistMixin, StyleMixin, AdvancedSettingsMixin):
```

Comment:
Let's keep the order of the Mixins. Put `AdvancedSettingsMixin` before `ArtMixin`.